src/main/java/org/tron/core/net/node/NodeImpl.java

```diff
@@ -84,6 +84,9 @@
   @Autowired
   private SyncPool pool;
 
+  @Autowired
+  private WitnessProductBlockService witnessProductBlockService;
+
   private MessageCount trxCount = new MessageCount();
 
   private Cache<Sha256Hash, TransactionMessage> TrxCache = CacheBuilder.newBuilder()
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "TrxCache" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S00116](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS00116))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1639&issues=AWalU-0plHK86Hco_1th&open=AWalU-0plHK86Hco_1th)

[ISSUE_KEY]: AWalU-0plHK86Hco_1th